static/js/jira-configuration.js

```diff
@@ -254,6 +255,27 @@ const handleModalDisplay = (title, info, type, data) => {
 		.attr("data-optional-modal-data", data.appId);
 }
 
+const additionalContent = (GHEServerUrl, appName) => {
+	let content = "";
+	if (!appName) {
+		// Get the list of all the apps within the GH Enterprise server
+		const apps = $(`.jiraConfiguration__enterpriseServer__header__container[data-server-baseurl='${GHEServerUrl}'] + .jiraConfiguration__enterpriseConnections > details`);
+		if ($(apps).length > 0) {
+			content += "Please make sure you delete these apps in GitHub too.<br/>";
```

Comment:
Do we want to run this message by a content designer?

Reply:
Yep agreed, will do that tomorrow!

Reply:
Confirmed with Matt and have updated the UI.